.github/ISSUE_TEMPLATE/website-issue.yaml

```diff
@@ -0,0 +1,45 @@
+name: Problems with the website
+description: Generate a report on an issue pertaining specifically to the website (https://community-scripts.github.io/ProxmoxVE/). For other inquires please share them in the Discussions section.
```

Comment:
Unfortunately Github does not support markdown links or anchors in template name or description fields